python-threatexchange/threatexchange/cli/match_cmd.py

```diff
@@ -29,6 +37,21 @@
 TMatcher = t.Callable[[pathlib.Path], t.List[IndexMatch]]
 
 
+@dataclass
+class _IndexMatchWithRotation(t.Generic[T]):
+    match: IndexMatchUntyped[SignalSimilarityInfo, T]
+    rotation_type: t.Optional[RotationType] = field(default=None)
+
+    def __str__(self):
+        # Supposed to be without whitespace, but let's make sure
+        distance_str = "".join(self.match.similarity_info.pretty_str().split())
+        if self.rotation_type is None:
+            return distance_str
+        else:
+            rotation_info = f" [{self.rotation_type.name}]"
+            return distance_str + rotation_info
```

Comment:
You can save on indenting by omitting the else.

You can also combinat the format string together since you are making another concat
```suggestion
        if self.rotation_type is None:
            return distance_str
        return f"{distance_str} [{self.rotation_type.name}]"
```